internal/app/mistapiconnector/mistapiconnector_app.go

```diff
@@ -501,8 +501,14 @@ func (mc *mac) sign(body string, sharedSecret string) string {
 	return signature
 }
 
-func (mc *mac) triggerUserNew(w http.ResponseWriter, r *http.Request, lines []string, rawRequest string) bool {
+func (mc *mac) webhookStatusUpdate(status livepeer.WebhookStatus, id string) {
+	statusRequestUpdate := map[string]interface{}{
```

Comment:
We should have proper request types for all these maps! Makes interfaces more maintainable and easier to use.